financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetState.kt

```diff
@@ -9,38 +11,22 @@ import com.stripe.android.financialconnections.model.FinancialConnectionsSession
  *  Class containing all of the data needed to represent the screen.
  */
 internal data class FinancialConnectionsSheetState(
+    val initialArgs: FinancialConnectionsSheetActivityArgs,
     val activityRecreated: Boolean = false,
-    val manifest: FinancialConnectionsSessionManifest? = null,
-    val authFlowActive: Boolean = false
-) {
+    @PersistState val manifest: FinancialConnectionsSessionManifest? = null,
+    @PersistState val authFlowActive: Boolean = false,
+    val viewEffect: FinancialConnectionsSheetViewEffect? = null
```

Comment:
Now viewEffects are part of the state, ensuring we commit to an Unidirectional Data Flow.

- Recommended readings about why effects should be on state: Recommended reading - https://developer.android.com/jetpack/guide/ui-layer/events#consuming-trigger-updates